transcom/mymove
# B 21435 int

## [B-21435](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1015238)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.



### How to test

1. run `make db_dev_fresh `or `make db_dev_migrate`
2. view db client or psql-dev to see that the following tables have been created: 
-re_states, re_us_post_region, re_oconus_rate_areas, re_intl_transit_times, 


### Database

#### Any new migrations/schema changes:


## Screenshots
